src/pages/[language]/[...slug]/index.query.graphql

```diff
@@ -82,6 +82,7 @@ query LandingPage($locale: SiteLocale, $slug: String) {
                   }
                 }
               }
+              variant
```

Comment:
Nitpick, would move this to line 76 just under `id` to make it easier to see what this field belongs to.